addons/account/models/account_move.py

```diff
@@ -1372,12 +1372,10 @@ def _check_balanced(self):
             sums = [res[1] for res in query_res]
             raise UserError(_("Cannot create unbalanced journal entry. Ids: %s\nDifferences debit - credit: %s") % (ids, sums))
 
-    def _check_fiscalyear_lock_date(self):
-        for move in self.filtered(lambda move: move.state == 'posted'):
-            lock_date = max(move.company_id.period_lock_date or date.min, move.company_id.fiscalyear_lock_date or date.min)
-            if self.user_has_groups('account.group_account_manager'):
-                lock_date = move.company_id.fiscalyear_lock_date
-            if move.date <= (lock_date or date.min):
+    def _check_fiscalyear_lock_date(self, only_posted=True):
```

Comment:
the paramerter `only_posted` will probably hide most of the messages that could arise (and defeat the purpose of this task).

I was about to say to default it to False, but actually I don't think we need it all